shell/shell.c

```diff
@@ -897,6 +897,7 @@ static void set_view_type(ShellViewType viewtype, gboolean reload)
 
     shell->normalize_percentage = TRUE;
     shell->view_type = viewtype;
+    shell->_order_type = 0;
```

Comment:
I'm curious why this fixes the issue: this is setting the order type to descending.